packages/extension/src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "version": "0.0.0",
-  "name": "daily.dev | Where Developers Grow Together",
+  "name": "daily.dev | Where developers grow together",
   "short_name": "daily.dev",
   "description": "Get a feed of the hottest developer news personalized to you.",
```

Comment:
Let's update `description` to: "daily.dev is a professional network for developers to learn, collaborate, and grow together."